ds_judgements_public_ui/sass/includes/govuk_overrides/_grid.scss

```diff
@@ -367,3 +398,103 @@
     padding-bottom: none;
   }
 }
+
+.govuk-grid-column-one-third__info_cards_home {
+  display: flex;
+  flex-direction: row;
+  gap: $space-4;
+
+  max-width: 24rem;
+  margin: $space-2;
+
+  background-color: colour-var("accent-background-light");
+  border-top: 4px solid $color-yellow;
+
+  &:hover {
+    text-decoration: none;
+    background-color: colour-var("accent-background");
+  }
+
+  @media (max-width: $grid-breakpoint-medium) {
+    min-width: 65%;
+    height: $space-6;
+    margin-left: 0;
+
+    background-color: transparent;
+    border-top: none;
+
+    &:hover {
+      background-color: transparent;
+    }
+  }
+}
+
+.govuk-grid-column-one-third__info_cards_home_image {
+  width: 111px;
+  height: 175px;
+
+  @media (max-width: $grid-breakpoint-medium) {
+    display: none;
+  }
+
+  @media (width <= 1199px) {
```

Comment:
Can we add a breakpoint variable for this?

Reply:
@jlhdxw this is specific for this so I did not realise we must add breakpoints variables as in the code base we already have some without that are specific as we may not always have variables which apply. So does this mean in codebase we will be changing all those that re not variables to variables now?